server/methods/getUsersOfRoom.js

```diff
@@ -44,9 +44,9 @@ Meteor.methods({
 			throw new Meteor.Error('error-not-allowed', 'Not allowed', { method: 'getUsersOfRoom' });
 		}
 
-		const total = Subscriptions.findByRoomIdWhenUsernameExists(rid).count();
+		const total = findUsers({ rid }).count();
```

Comment:
But this will occasionate the same problem, isn't it? 🤔 because the counter of this query and the one on L49 will be different (since this line is just filtering by rid, not by status)